addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -168,9 +169,9 @@
                             <field name="move_id" invisible="1"/>
                         </tree>
                     </field>
-                    <h4 attrs="{'invisible': [('finished_workorder_line_ids', '=', [])]}">Finished Products</h4>
-                    <field name="finished_workorder_line_ids" attrs="{'invisible': [('finished_workorder_line_ids', '=', [])], 'readonly': [('state', 'in', 'cancel', 'done')]}">
-                        <tree editable="bottom" create="0" delete="0">
+                    <h4 groups="mrp.group_mrp_byproducts">Finished Products</h4>
```

Comment:
nope, if you don't enable the byproducts and you want to change some finished lots you will be stuck